code/datums/antagonists/datum_devil.dm

```diff
@@ -387,8 +363,8 @@ var/global/list/lawlorify = list (
 						return 1
 				return 0
 
-/datum/devilinfo/proc/hellish_resurrection(mob/living/body)
-	message_admins("[owner.name] (true name is: [truename]) is resurrecting using hellish energy.</a>")
+/datum/antagonist/devil/proc/hellish_resurrection(mob/living/body)
+	message_admins("[owner.mind.name] (true name is: [truename]) is resurrecting using hellish energy.</a>")
```

Comment:
I think you might need to store the mind you expect to have in a var, because this proc won't work with how antag datums are set up; they assume that if you have no mob, you don't fucking exist at all(and someone being deleted will lose their antag datums)